js/index.js

```diff
@@ -0,0 +1,210 @@
+'use strict'
+
+const blinding = (iter, on = null) => {
+    const action = on == null ?
+        "toggle" :
+        on ?
+            "add" :
+            "remove"
+
+    for (const item of iter) {
+        item.classList[action]('blind');
+    }
+};
+
+const toHtml = source => {
+    let $div = document.createElement("div");
+    $div.innerHTML = source;
+    return $div.firstElementChild;
+}
+
+class TodoApp {
+    constructor() {
+        this.$todoInput = document.getElementById("new-todo-title");
+        this.$todoList = document.getElementById("todo-list");
+        this.$todoFooter = document.querySelector(".count-container");
+        this.previous = "";
+        this.build();
+        this.eventRegist();
+    }
+
+    getTodoEntity(idx) {
+        let $todoItem = this.$todoList.querySelector(`[data-idx='${idx}']`);
+        return {
+            idx:Number(idx),
+            value:$todoItem.querySelector('.label').textContent,
+            check:$todoItem.querySelector('.toggle').checked
+        }
+    }
+
+    getLastIndex() {
+        let todoItems = lsg.get('todoItems');
+        let max = 0;
+        if (!todoItems || todoItems.length === 0) return max;
+
+        for (const item of todoItems) {
+            if (max < item.idx) max = item.idx;
+        }
+        return max;
```

Comment:
```suggestion
        const todoItems = lsg.get('todoItems');
        if (!todoItems || todoItems.length === 0) return 0;
        return todoItems.reduce((max, item) => Math.max(max, item.idx));
```
이렇게 표현할 수 있습니다!
혹은 그냥 lastIndex를 멤버변수로 관리해보세요.
add 할 때마다 +1을 해주는거죠

Reply:
아 저런 방법이 있군요..! Array의 reduce기능은 생각 못했네요